staubli_rx160_gazebo/urdf/rx160.xacro

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<robot name="staubli_rx160_gazebo" xmlns:xacro="http://ros.org/wiki/xacro">
```

Comment:
> Edit: I'd like to keep line 1; gedit fails to highlight the syntax otherwise, and that is quite annoying.

I don't really understand? I did not suggest removing the xml preamble. Just the comment `<!-- RX160 -->`.